setup.py

```diff
@@ -35,10 +42,10 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
```

Comment:
Should also update the README to say we support 3.5+ (currently says 3.3+).

Reply:
Done.